testing/python/amd/test_tilelang_gfx950_copy_async.py

```
@@ -0,0 +1,230 @@
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _is_gfx950() -> bool:
+    try:
+        from tilelang import tvm
+
+        mcpu = str(tvm.target.Target("rocm").attrs.get("mcpu", ""))
+        return "gfx950" in mcpu
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)